spec/integration/endpoints/steps_spec.rb

```diff
@@ -123,6 +127,57 @@ def create_step(step)
           end
 
         end
+
+        context 'add_step functionality' do
+          let(:testcase_addstep) {
+            {
+              'name'      => 'stepName2',
+              'classname' => 'caseName2',
+              'result'    => 'success',
+              'test_data' => { 'any_content' => 'xxx' },
+              'duration' => 56
+            }
+          }
+          let(:cache) {
+            Travis::GuestAPI::Cache.new 100 , Travis.config.redis
+          }
+          let(:sample_record) {
+            {
+              'name' => 'foo',
+              'classname' => 'caseName2',
```

Comment:
I think this shows a bit of issue in naming convention. When we have class_position, then why not class_name as well?